pages/_middleware.ts

```diff
@@ -1,23 +1,51 @@
 import { NextRequest, NextResponse } from 'next/server'
 
-const BLOCKED_COUNTRIES = ['US', 'BY', 'CU', 'IR', 'IQ', 'CI', 'LR', 'KP', 'SD', 'SY', 'ZW', 'CA']
-const ALLOWED_URLS = RegExp(
-  '(/images|/favicon.ico|/font|/api/auction/getLatestAuction|/api/auction/getAuctionById|/api/auction/getLastHedge)',
-)
-
-export function middleware(req: NextRequest) {
-  const country = req?.geo?.country
-  const response = NextResponse.next()
-  const path = req.url
-
-  if (country && BLOCKED_COUNTRIES.includes(country) && !ALLOWED_URLS.test(path)) {
-    return NextResponse.rewrite('/blocked')
+import { redis } from '../utils/redisClient'
+import { isVPN } from '../utils/vpn'
+
+const ignoredPaths = ['/api', '/favicon.ico', '/static', '/_next', '/blocked']
```

Comment:
We dealing with /api in other PR? Can discuss if any qs

Reply:
done in this commit: https://github.com/KMKoushik/squeeth-portal/pull/118/commits/09c56044f8809bd44a501de437af13252193579e